### PR TITLE
Armstrong checker

### LINKED ISSUE #1418: Armstrong checker
**Is your feature request related to a problem? Please describe.**
The background color of the page and the input box is same which is not so attractive.

**Describe the solution you'd like**
If the colors are different it would look better.

**Describe alternatives you've considered**
Changing the background color of the page.

**Additional context**
Screenshot (53)

@TusharKesarwani @zugzwang03 please assign me this issue

### PR DESCRIPTION
**Pull request by suma30**
## 🛠️ Issue (Number)

Closes #1418 

# Description

Changed the background color of the site and also the color of the main heading which is appealing now

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] I have made this from my own
- [ ] I have taken help from some online resourses 
- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] The title of my pull request is a short description of the requested changes.

@TusharKesarwani @zugzwang03 @harshit-paneri Please review my PR for this armstrong checker site

### CODE REVIEW

**Review by harshit-paneri — COMMENTED** [review]
Deploy Link?

**Review by zugzwang03 — APPROVED** [review]
extremely minor change though

**Comment by suma30** [comment]
@TusharKesarwani Please merge this PR for Armstrong checker

File: Projects/Blog Project Website/contact.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h2>Feel Free to Contact Us</h2>
                     <input type="text" placeholder="Enter Your Email Id">
                 </div>
                 <div class="form-box">
-                    <textarea name="" id="" cols="30" rows="10" placeholder="How may we help you?"></textarea>
+                    <textarea name="" id="" cols="30" rows="8" placeholder="How may we help you?"></textarea>
```
Comment (TusharKesarwani) [inline Projects/Blog Project Website/contact.html:50]: Do not perform updates in another project

**Pull request closed without merging**